### ISSUE TITLE
[Bug Report][2.5.3] VSelectList does not provide aria-label or aria-labelledby

### ISSUE THREAD

**Issue by laurentupa** [vuetifyjs/vuetify#13791]
### Environment
**Vuetify Version:** 2.5.3
**Vue Version:** 2.6.12
**Browsers:** Chrome 91.0.4472.101
**OS:** Windows 10

### Steps to reproduce
1) Open dropdown (VSelect)
2) Inspect the list of items (VSelectList)

### Expected Behavior
There is either an aria-label or aria-labelledby attribute on the VSelectList element that contains role="listbox".

When you inspect the list of items this is what I'd expect to see: 
`<div
  role="listbox"
  tabindex="-1"
  class="v-list v-select-list v-sheet theme--light theme--light"
  aria-label="some-label-here"
  id="list-3">`

### Actual Behavior
Neither aria-label nor aria-labelledby exist on the element.

When you inspect the list of items this is what is seen: 
`<div
  role="listbox"
  tabindex="-1"
  class="v-list v-select-list v-sheet theme--light theme--light"
  id="list-3">`

### Reproduction Link
<a href="https://codepen.io/laurentupa/pen/NWpeWvB" target="_blank">https://codepen.io/laurentupa/pen/NWpeWvB</a>

### Other comments
It looks like the issue is [here](https://github.com/vuetifyjs/vuetify/blob/v2.5.3/packages/vuetify/src/components/VSelect/VSelectList.ts#L258). I believe this is an issue based on WAI-ARIA rules.

**Comment by KaelWD** [comment]
What do you think the label should be? It's already referenced by `aria-owns` on the select itself.

**Comment by laurentupa** [comment]
After reading the [WAI-ARIA Authoring Practices for listbox](https://www.w3.org/TR/wai-aria-practices-1.1/#listbox_roles_states_props) again, I came across this rule, which states:
> If the listbox is not part of another widget, then it has a visible label referenced by aria-labelledby on the element with role listbox.

So, because I think the listbox is part of another widget, I believe I may be mistaken in saying that an aria label (or labelled-by) attribute should be on the element. I think this is an axe-core issue.

**Comment by KaelWD** [comment]
It's possible. This has been tested to work with NVDA and android TalkBack so the current implementation is fine in real scenarios.

**Comment by lalwanikamaldev** [comment]
Any update on this? i am also getting issue from accessibility Tool for dropdown in table 
![image](https://user-images.githubusercontent.com/65231431/181257162-6fb98e34-0596-4a82-aa4c-cfe3749b8466.png)
![image](https://user-images.githubusercontent.com/65231431/181257488-d55f67ce-3e13-417d-8506-6614ec4e9f31.png)
